<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>751</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="stockToFind"/>
       </item>
       <item>
        <widget class="QPushButton" name="findButton">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="companyName">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Company Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="price_lab">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Price:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showPrice">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <property name="spacing">
        <number>40</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="open">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Open:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showOpen">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="preClose">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Previous Close:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showPreClose">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="dayRange">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Day's Range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showDayRange">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="avgVol">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Avg. Volume:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showAvgVol">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="w52">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>52 Week Range:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showWeekRange">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QLabel" name="vol">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Volume:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="showVolume">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>N/A</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
